test/test_run_analysis.rb

```diff
@@ -316,7 +316,7 @@ def test_national_baseline
     assert(File.exist?(cli_output_log))
     cli_output = File.readlines(cli_output_log)
     _assert_and_puts(cli_output, 'ERROR', false)
-    _verify_outputs(cli_output_log)
+    _verify_outputs(cli_output_log, true)
```

Comment:
Why is anything in this file changed?

Reply:
Oh. This is because we added `register_logs(runner, new_runner)` to the meta measures?

Reply:
Yep!